app/models/job/Job.scala

```diff
@@ -286,6 +287,18 @@ class JobService @Inject()(wkConf: WkConf,
     ()
   }
 
+  def cleanUpIfFailed(job: Job): Fox[Unit] =
+    if (job.state == JobState.FAILURE && job.command == "convert_to_wkw") {
```

Comment:
I could have created a more nested version that is easier to extend, but I remembered the XP practice of code for today, not tomorrow and this should not be complicated to change in the future if more jobs need such a functionality.